zend/callable.cpp

```diff
@@ -99,10 +102,10 @@ void Callable::initialize(zend_function_entry *entry, const char *classname, int
         _argv[_argc + 1].class_name = reinterpret_cast<const char*>(this);
 #else
         // @todo    this is broken. the zend engine, from 7.2 onwards copies over
-        //          the struct and slices of the last element, because the num_args
+        //          the struct and slices off the last element, because the num_args
         //          is incorrect in their view. another place to put this may be
         //          hiding it behind the fname
-        _argv[_argc + 1].type = reinterpret_cast<zend_type>(this);
+       // _argv[_argc + 1].type = reinterpret_cast<zend_type>(this);
```

Comment:
I couldn't solve this - I don't know what functionality breaks by not fixing it. Its related to the zend_type becoming a struct